### PR TITLE
feat(ci): generate robots.txt for test.leapdao.org

### LINKED ISSUE #41: make test.leapdao.org non-indexable for search engines
# Bounty
the page can lead to duplicate content penalties by google, or other confusions.

## Scope

- prohibit bot indexing, or find other way to deal with availability (password protection?)

## Deliverables

- only leapdao.org should be indexed
  - check on google, Bing, and yandex

## Roles
bounty gardener: @johannbarbie  / 20%
bounty worker: name / 60%
bounty reviewer: name / 20%

### PR DESCRIPTION
**Pull request by troggy**
Resolves #41

### CODE REVIEW

**Comment by troggy** [comment]
Check with https://sitechecker.pro/robots-tester/

File: .travis.yml [hunk 1]
```diff
@@ -36,5 +36,7 @@ deploy:
 
 after_deploy:
   - aws configure set preview.cloudfront true
+  - "echo \"User-agent: *\n\rDisallow: /\n\r\" > robots.txt"
+  - "if [ \"$TRAVIS_BRANCH\" != \"master\" ]; then aws s3 cp robots.txt s3://$TEST_S3_BUCKET/ --acl public-read; fi"
```
Comment (johannbarbie) [inline .travis.yml]: how about including `aws cloudfront create-invalidation --distribution-id $TEST_CLOUDFRONT_DISTRIBUTION --paths` in that if statement?

Comment (troggy) [reply .travis.yml]: yes. this would be better. Tho out of scope

Comment (troggy) [reply .travis.yml]: addressed

**Pull request merged**